test/async-hooks/test-tlswrap.js

```diff
@@ -4,6 +4,7 @@ const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
+const path = require('path');
```

Comment:
This is a nit pick, but can you move `path` below `assert`? In tests, we try to (but admittedly often don't) to keep the built-in modules in alphabetical order. (Well, ASCII order actually. https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#lines-7-8)